src/actions/rotate/Rotate.ts

```diff
@@ -0,0 +1,47 @@
+import horizontalFlip from "./horizontalFlip";
+import verticalFlip from "./verticalFlip";
+import angle from "./angle";
+import Action from "../Action";
+import {IAction} from "../../interfaces/IAction";
+import Param from "../../parameters/Param";
+const HORIZONTAL_FLIP = 'hflip';
+const VERTICAL_FLIP = 'vflip';
+const PARAM_KEY = 'a';
```

Comment:
PARAM_KEY is a param-level variable, a single action can have multiple params, so I'm not sure this is the right approach

Reply:
Rotate is a single param action, future refactor can group those under SingleParamAction

Reply:
This feels like a detail though, especially as this is completely out of our control whether something is a single or multi param.

This adds constraints we don't necessarily want and that we might eventually regret. 

Once this becomes a multiParam action, a lot needs to change in this Action.

Reply:
please suggest different solution

Reply:
We'll PR it once this is merged as this is an internal detail that doesn't affect the feature itself.